frontend/src/empezar-reunion/EmpezarReunion.js

```diff
@@ -12,13 +13,31 @@ class EmpezarReunion extends React.Component {
     super(props);
     this.state = {
       redirect: false,
+      cargando: false,
     };
   }
 
   handleEmpezarReunion = () => {
-    backend.empezarReunion().then(() => {
-      this.setState({ redirect: true });
-    });
+    this.setState({ cargando: true });
+    setTimeout(this.requestEmpezarReunion, 3000);
+  };
+
+  requestEmpezarReunion = () => {
+    backend.empezarReunion().then(() => { this.setState({ redirect: true }); })
+      .catch(() => {
+        this.setState({ cargando: false });
+        alert('Error al iniciar reunión!');
+        return <Redirect to="/" />;
+      });
+  }
+
+  iniciarReunion = () => {
```

Comment:
El nombre del metodo no me dá la idea de que esto renderiza algo, parece más una acción.
Algo como:
```suggestion
  BotonParaIniciarReunion = () => {
```


De paso, si está en mayusculas, podemos `JSX`earlo y tenes más abajo:
```
<TitleAndButton>
  <Title>No hay ninguna reunión activa</Title>
  <BotonParaIniciarReunion />
</TitleAndButton>
```

Reply:
Tengo un debate interno si quiero que eso esté en un archivo aparte de "componentes" o no.
Creo que por ahora diría que no; pero estemos atentos a este tipo de cosas que podamos reutilizar.

(un botón que a veces está cargando, suena a algo que vamos a reutilizar)

Reply:
Ya lo pasamos a un componente.